pkg/network/tcp_peer.go

```diff
@@ -27,6 +27,7 @@ const (
 )
 
 var (
+	errGone           = errors.New("the peer has gone already")
```

Comment:
probably `is gone`

Reply:
That was my thought initially, but for some reason I've changed my mind and phrased it liked this. Wrong: https://english.stackexchange.com/questions/41453/she-is-gone-versus-she-has-gone#41454